docs/do-not-merge.md

```diff
@@ -0,0 +1,827 @@
+* You can now speed up `kill` tasks by batch deleting multiple segments stored in S3 [#14131](https://github.com/apache/druid/pull/14131)
+* Kill tasks that delete unused segments now publish a task report containing kill stats such as `numSegmentsKilled`, `numBatchesProcessed`, and `numSegmentsMarkedAsUnused` [#15023](https://github.com/apache/druid/pull/15023)
+* `IndexerSQLMetadataStorageCoordinator` now uses the JDBI `PreparedBatch` instead of issuing single update statements inside a transaction to mitigate scaling challenges [#14639](https://github.com/apache/druid/pull/14639)
+
+## Web console
+
+### Added UI support for segment loading query context parameter
+
+The web console now supports the `waitUntilSegmentsLoad` query context parameter.
+
+![UI for waitUntilSegmentsLoad context parameter](image.png)
+
+[#15110](https://github.com/apache/druid/pull/15110)
+
+### Added concurrent compaction switches
+
+The web console now includes concurrent compaction switches.
+
+The following screenshot shows the concurrent compaction switch in the classic batch ingestion wizard:
+![Classic batch ingestion wizard](image-1.png)
+
+The following screenshot shows the concurrent compaction switch in the compaction configuration UI:
+![Compaction configuration UI](image-2.png)
+
+[#15114](https://github.com/apache/druid/pull/15114)
+
+### Other web console improvements
+
+* You can select to allow concurrent APPEND tasks in the web console [#15114](https://github.com/apache/druid/pull/15114)
+* You can now copy query results from the web console directly to the clipboard [#14889](https://github.com/apache/druid/pull/14889)
+* The web console now shows the execution dialog for `query_controller` tasks in the task view instead of the generic raw task details dialog. You can still access the raw task details from the ellipsis (...) menu [#14930)](https://github.com/apache/druid/pull/14930)
+* You can now select a horizontal range in the web console time chart to modify the current WHERE clause [#14929](https://github.com/apache/druid/pull/14929)
+* You can now set dynamic query parameters in the web console [#14921](https://github.com/apache/druid/pull/14921)
+* You can now edit the Coordinator dynamic coonfiguration in the web console [#14791](https://github.com/apache/druid/pull/14791)
+* You can now prettify SQL queries and use flatten with a Kafka input format [#14906](https://github.com/apache/druid/pull/14906)
+* A warning now appears when a CSV or TSV sample contains newlines that Druid does not accept [#14783](https://github.com/apache/druid/pull/14783)
+* You can now select a format when downloading data [#14794](https://github.com/apache/druid/pull/14794)
+* Improved the clarity of cluster default rules in the retention dialog [#14793](https://github.com/apache/druid/pull/14793)
+* Enabled Kafka multi-topic ingestion from the data loader [#14833](https://github.com/apache/druid/pull/14833)
```

Comment:
This maybe deserves its own H3 since the feature it's supporting has its H3 like concurrent append and replace does.

Reply:
Moved id 14833 to H3